listenbrainz/webserver/static/js/src/listens/ListenCard.tsx

```diff
@@ -307,18 +347,30 @@ export default class ListenCard extends React.Component<
               </ul>
             </>
           )}
-          <button
-            title="Play"
-            className="btn-transparent play-button"
-            onClick={this.playListen}
-            type="button"
-          >
-            {isCurrentlyPlaying ? (
-              <FontAwesomeIcon size="1x" icon={faPlay as IconProp} />
-            ) : (
-              <FontAwesomeIcon size="2x" icon={faPlayCircle as IconProp} />
-            )}
-          </button>
+          {disablePlay ? null : (
```

Comment:
This is a potentially useful feature, but I have 2 comments:
1. You also need to check disablePlay on line 276 (`onDoubleClick={this.playListen}`)
2. Why don't we want this page to be playable?